### PR TITLE
New page for HTMLInputElement.indeterminate

### PR DESCRIPTION
**Pull request by Josh-Cena**
https://github.com/mdn/content/issues/35544

### CODE REVIEW

File: files/en-us/web/api/htmlinputelement/indeterminate/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "HTMLInputElement: indeterminate property"
+short-title: indeterminate
+slug: Web/API/HTMLInputElement/indeterminate
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.indeterminate
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`indeterminate`** property of the {{domxref("HTMLInputElement")}} interface returns a boolean value that indicates whether the checkbox is in the _indeterminate_ state. For example, a "select all/deselect all" checkbox may be in the indeterminate state when some but not all of its sub-controls are checked. It is unrelated to the {{domxref("HTMLInputElement.checked")}} property, and an indeterminate checkbox can be either checked or unchecked. The `indeterminate` state can only be set via JavaScript, and only affects the checkbox's appearance, not its value when submitted.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/indeterminate/index.md:11]:
```suggestion
The **`indeterminate`** property of the {{domxref("HTMLInputElement")}} interface returns a boolean value that indicates whether the checkbox is in the _indeterminate_ state. For example, a "select all/deselect all" checkbox may be in the indeterminate state when some but not all of its sub-controls are checked. The `indeterminate` state can only be set via JavaScript and is only relevant to [`checkbox`](/en-US/docs/Web/HTML/Element/input/checkbox) controls.

It is unrelated to the {{domxref("HTMLInputElement.checked")}} property, and an indeterminate checkbox can be either checked or unchecked. Being indeterminate only affects the checkbox's appearance (see example below), not its value when submitted (which is controlled by the checkedness).
```

File: files/en-us/web/api/htmlinputelement/indeterminate/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- {{domxref("HTMLInputElement")}}
+- {{domxref("HTMLFormElement.checked")}}
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/indeterminate/index.md:44]:
```suggestion
- {{domxref("HTMLInputElement.checked")}}
```

File: files/en-us/web/api/htmlinputelement/indeterminate/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+slug: Web/API/HTMLInputElement/indeterminate
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.indeterminate
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`indeterminate`** property of the {{domxref("HTMLInputElement")}} interface returns a boolean value that indicates whether the checkbox is in the _indeterminate_ state. For example, a "select all/deselect all" checkbox may be in the indeterminate state when some but not all of its sub-controls are checked. The `indeterminate` state can only be set via JavaScript and is only relevant to [`checkbox`](/en-US/docs/Web/HTML/Element/input/checkbox) controls.
+
+It is unrelated to the {{domxref("HTMLInputElement.checked")}} property, and an indeterminate checkbox can be either checked or unchecked. Being indeterminate only affects the checkbox's appearance (see example below), not its value when submitted (which is controlled by the checkedness).
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/indeterminate/index.md:13]:
```suggestion
It is unrelated to the {{domxref("HTMLInputElement.checked")}} property, and an indeterminate checkbox can be either checked or unchecked. Being indeterminate only affects the checkbox's appearance (see example below), not its `checked` state.
```
the name and value pair are either submitted or not; the checkedness doesn't control the `value` per se.

**Review by estelle — APPROVED** [review]

File: files/en-us/web/api/htmlinputelement/indeterminate/index.md (hunk 3 above)
Comment (Josh-Cena) [reply files/en-us/web/api/htmlinputelement/indeterminate/index.md:13]:
```suggestion
It is unrelated to the {{domxref("HTMLInputElement.checked")}} property, and an indeterminate checkbox can be either checked or unchecked. Being indeterminate only affects the checkbox's appearance (see example below), not its presence when submitted (which is controlled by the checkedness).
```

What about this?

**Pull request merged**